Header header
float32[] data
int16 width
int16 height
int16 lam
float32[] central_wavelengths
float32[] qe
float32[] fwhm_nm
